plugins/uefi-capsule/fu-uefi-capsule-plugin.c

```diff
@@ -645,6 +646,16 @@ fu_uefi_capsule_plugin_get_default_esp(FuPlugin *plugin, GError **error)
 				g_warning("failed to mount ESP: %s", error_local->message);
 				continue;
 			}
+			/* if user specified, make sure that it matches */
```

Comment:
total nit, but you need a space before the comment line

Reply:
fixed